packages/@sanity/desk-tool/src/panes/document/statusBar/sparkline/ReviewChangesButton/AnimatedStatusIcon.tsx

```diff
@@ -0,0 +1,176 @@
+import React, {ComponentProps} from 'react'
+import {motion} from 'framer-motion'
+import styled, {keyframes} from 'styled-components'
+
+const StyledMotionPath = styled(motion.path)`
+  transform-origin: center;
+`
+
+type MotionCircleProps = Omit<ComponentProps<typeof motion.circle>, 'd'>
+type MotionPathProps = Omit<ComponentProps<typeof motion.path>, 'd'>
+
+const Circle = (props: MotionCircleProps) => (
+  <motion.circle fill="none" r="8" cx="12.5" cy="12.5" strokeWidth="1.2" {...props} />
+)
+const Arrows = (props: MotionPathProps) => (
+  <StyledMotionPath
+    fill="none"
+    d="M14 17.5619L11.5 20.5L14.5 23.0619M11 7.43811L13.5 4.50001L10.5 1.93811"
+    {...props}
+  />
+)
+const Checkmark = (props: MotionPathProps) => (
+  <motion.path d="M9.5 12.1316L11.7414 14.5L16 10" {...props} />
+)
+const Edit = (props: MotionPathProps) => (
+  <motion.path d="M15 7L18 10M6 19L7 15L17 5L20 8L10 18L6 19Z" {...props} />
+)
+
+const rotateAnimation = keyframes`
+  0% {
+    transform: rotate(0);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+`
+const RotateGroup = styled.g`
+  transform-origin: center;
+
+  &[data-rotate] {
+    animation: ${rotateAnimation} 1s ease-in-out infinite;
+  }
+`
+
+const root = {
+  syncing: {
+    scale: 1,
+    transition: {
+      duration: 0,
+    },
+  },
+  saved: {
+    scale: [1, 0.8, 1.3, 0.9, 1.15, 1.1, 1],
```

Comment:
I've exprimented a bit with the timing here, and think the following keyframes work sliiightly better 
` scale: [1, 0.8, 1.2, 0.9, 1.1, 0.95, 1.05, 0.99, 1]`

Reply:
Ok, thanks! I'll update :)